packages/react-change-highlight/src/ShadowChild.tsx

```diff
@@ -35,23 +37,26 @@ const ShadowChild: React.FC<Props> = ({
   }
 
   useEffect(() => {
-    if (initialMount?.current) {
+    if (!newlyAddedOnly && initialMount?.current) {
```

Comment:
:clap: :clap: 
I couldn't figure it out back then, but it seems very silly :sweat_smile: